extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -470,12 +471,12 @@ public function getArticleMatchForTermAndNamespaces( $term, array $namespaces )
 		$title = $searchEngine->getNearMatch( $term );
 		$articleId = ( $title !== null ) ? $title->getArticleId() : 0;
 		if ( ( $articleId > 0 ) && ( in_array( $title->getNamespace(), $namespaces ) ) ) {
-			$page = $this->getPageFromPageId( $articleId );
-			if ( $page instanceof \Article ) {
+			try {
+				$page = $this->getPageFromPageId( $articleId );
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "$page" local variable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)